services/movie.py

```diff
@@ -0,0 +1,44 @@
+from django.db.models.query import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    if not genres_ids and not actors_ids:
+        return Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        films_with_genres = Movie.objects.filter(genres__id__in=genres_ids)
+        return films_with_genres.filter(actors__id__in=actors_ids)
+
+    if genres_ids:
+        return Movie.objects.filter(genres__id__in=genres_ids)
+
+    return Movie.objects.filter(actors__id__in=actors_ids)
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(
+        movie_title: str,
+        movie_description: str,
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]